packages/oauth/src/twitch.ts

```diff
@@ -93,7 +101,9 @@ class Twitch<A extends Auth> implements OAuthProvider<A> {
 			createUser,
 			existingUser,
 			providerUser: twitchUser,
-			accessToken
+			accessToken,
+			refreshToken,
+			expiresIn
```

Comment:
What's `expiresIn`? If it's the access token, I think it'd be more clear if it was `accessTokenExpiresIn`.

Reply:
I'm pretty sure for every oauth2 provider the `expires_in` prop is always referring to the `access_token` expiration. This'll have to be changed with the other oauth providers as well.

Reply:
Ah, ok, so it's part of the spec (though optional): https://www.rfc-editor.org/rfc/rfc6749#section-5.1